project/ProjectPlugin.scala

```diff
@@ -17,6 +17,7 @@ object ProjectPlugin extends AutoPlugin {
     lazy val V = new {
       val avrohugger: String          = "1.0.0-RC22"
       val circe: String               = "0.13.0"
+      val hammock                     = "0.10.0"
```

Comment:
Hammock `0.10.0` is bringing _circe_ `0.12.1`, as you can see [here](https://github.com/pepegar/hammock/blob/v0.10.0/build.sbt#L22). However, that will provoke a collision with the circe version we are using in this build: 0.13.0.

So we have two options: my suggestion would be using an `http4s` client, as it is evolving at the same time as typelevel ecosystem does.

Reply:
ready, please take a look @juanpedromoreno